examples/common/entry/entry_html5.cpp

```diff
@@ -107,12 +113,17 @@ namespace entry
 			fullscreenStrategy.canvasResizedCallback = canvasResizeCb;
 			fullscreenStrategy.canvasResizedCallbackUserData = this;
 
-			emscripten_request_fullscreen_strategy("#canvas", false, &fullscreenStrategy);
+			emscripten_request_fullscreen_strategy(HTML5_TARGET_CANVAS_SELECTOR, false, &fullscreenStrategy);
 
 			emscripten_set_focus_callback(EMSCRIPTEN_EVENT_TARGET_DOCUMENT, this, true, focusCb);
 			emscripten_set_focusin_callback(EMSCRIPTEN_EVENT_TARGET_DOCUMENT, this, true, focusCb);
 			emscripten_set_focusout_callback(EMSCRIPTEN_EVENT_TARGET_DOCUMENT, this, true, focusCb);
 
+			bgfx::PlatformData pd;
+			bx::memSet(&pd, 0, sizeof(pd) );
+			pd.nwh = (void*)HTML5_TARGET_CANVAS_SELECTOR;
+			bgfx::setPlatformData(pd);
```

Comment:
This fixes a bug where in newer Emscriptens one cannot initialize canvas with a `NULL` target, but one must always specify the target canvas element to initialize on.